tests/integration/test_profiles_connection.py

```diff
@@ -1,13 +1,14 @@
 import pytest
 import ftplib
-import warnings
-import src
 
+import src
+from src import utils
 
 @pytest.mark.order(3)
 @pytest.mark.integration
 def test_profiles_connection() -> None:
-    config = src.types.Config.load()
+    config_path = utils.environment.get_config_path()
+    config = src.types.Config.load(config_path)
```

Comment:
This code does not have to change.

In the codebase, wherever you do:

```python
config = src.types.Config.load(config_path)
```

you do this before:

```python
config_path = utils.environment.get_config_path()
```

Please make the config path a default argument, so that you only have to pass it when it is different than `utils.environment.get_config_path()`.

Reply:
I think you mean that you want `Config.load` to take the value of `utils.environment.get_config_path()` as a default argument for `path`, i.e.:
```
default_config_path = utils.environment.get_config_path()
def load(
        path: str = default_config_path,
        ignore_path_existence: bool = False,
    ) -> Config:
```
so that we don't need to normally pass an argument to `Config.load`. Is that correct?

The problem I encounter when importing `utils/environmeny.py` into `types/config.py`: there is a circular import between the `types` and `utils` modules. i.e. `utils/report.py` imports `types`. I am not sure how to alleviate this issue in the pythonic import setup for this project. Do you have some idea?

One option is to add the function logic for `utils.environment.get_config_path()` directly inside the `Config` class, however I don't like this solution because we then need to define the environment key in multiple places (its needed for some tests and the config directory is also used in some places when metadata is instantiated) and it becomes hard to track its value. I don't think not modifying the function calls justified this.

Please let me know how you think we should handle this?

Reply:
That is true.

I think, the most elegant solution here would be to move the two functions from `utils/environment.py` into the `Config` class and remove the environment module.